packages/manager/modules/pci/src/projects/new/payment/index.js

```diff
@@ -29,6 +30,7 @@ angular
   .config(routing)
   .run(/* @ngTranslationsInject:json ./translations */)
   .component(component.name, component)
+  .service('PciProjectsService', PciProjectsService)
```

Comment:
Already defined in https://github.com/ovh/manager/blob/master/packages/manager/modules/pci/src/projects/projects.module.js#L40

Reply:
True, removed.
Thanks